tools/SwaggerCI/init.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -i
+
+# This script is created to set up the env to generate Azure PowerShell modules based on swagger. 
+
+# Installed required packages
+printenv
+sudo apt-get update \
```

Comment:
Suggest to use https://www.npmjs.com/package/n to reduce noise and speedup.

N_PREFIX=$HOME/.n
PATH=$N_PREFIX/bin:$PATH
npx n 14.15.5

export PATH and N_PREFIX then

Reply:
Thanks for recommendation. Currently we use NVM in autorest.powershell projects across different OS. I will try to play with N for a while and then consider whether to replace NVM with N in the future.